packages/desktop-client/src/components/transactions/TransactionsTable.jsx

```diff
@@ -521,12 +521,11 @@ function PayeeCell({
               style: inputStyle,
             }}
             showManagePayees={true}
-            tableBehavior={true}
+            clearOnBlur={false}
             focused={true}
-            onUpdate={onUpdate}
+            onUpdate={(id, value) => onUpdate?.(value)}
             onSelect={onSave}
             onManagePayees={() => onManagePayees(payeeId)}
-            isCreatable
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏same problem here. Would this lead to a regression?